superset-frontend/src/SqlLab/components/AceEditorWrapper/AceEditorWrapper.test.tsx

```diff
@@ -83,6 +73,9 @@ describe('AceEditorWrapper', () => {
   });
 
   it('renders sql from unsaved change', () => {
+    // TODO: need to rewrite the test because we retrieve 'currentQueryEditor' in SqlEditor
```

Comment:
@EugeneTorap Can you rewrite the test? It's important to have a test case for this scenario.

Reply:
@michael-s-molina Can I do it in the next PR?